obfx_modules/companion-legacy/inc/hestia/inc/sections/hestia-team-section.php

```diff
@@ -224,13 +223,16 @@ function hestia_team_content( $hestia_team_content, $is_callback = false ) {
 				foreach ( $icons_decoded as $value ) {
 					$social_icon = ! empty( $value['icon'] ) ? apply_filters( 'hestia_translate_single_string', $value['icon'], 'Team section' ) : '';
 					$social_link = ! empty( $value['link'] ) ? apply_filters( 'hestia_translate_single_string', $value['link'], 'Team section' ) : '';
-					
 					if ( ! empty( $social_icon ) ) {
+						$path = wp_parse_url( $social_link, PHP_URL_PATH );
+						$path = strstr( $path, '.', true );
+						$aria_label = sprintf( 'View the %s profile of %s', $path, $title );
+						$aria_label = apply_filters( 'hestia_team_social_media_aria_text', $aria_label );
```

Comment:
@girishpanchal30 why is the filter required for every icon, inside the loop, since the text is global and there is no way to target per icon since we don't pass that to the filter args? 

I would either move it before foreach or pass path, path and title to the filter. Moreover, have you validated what happens on when `wp_parse_url` returns false?

Reply:
@selul I have added a filter if the user wishes to filter aria-label text. Please review my latest changes and let me know if you have any problem

Reply:
lovely now ❤️